less.js

```diff
@@ -97,7 +98,27 @@ exports.translate = function(load) {
 			var p = Promise.resolve(
 				lessEngine.render(load.source, renderOptions)
 			);
-			return p.then(done);
+			return p.then(done).then(null, function(err){
+				// 404.
+				//!steal-remove-start
+				if(err.type === "File" && /404/.test(err.message)) {
+					if(loader._addSourceInfoToError) {
+						var fn = err.filename.split("/").pop();
+						var msg = "The stylesheet [" + fn + "] was able to fetch a dependency.\n" +
```

Comment:
Should “was able” be “wasn’t able”?

Reply:
yep, nice catch

Reply:
Yay for including screenshots in your PR (also yay for this PR in general)!